031-tls-encrypt-the-kafka-connect-rest-api.md

```diff
@@ -0,0 +1,168 @@
+# TLS encrypting the Kafka Connect REST API
+
+This proposal adds the option to reduce access to the Kafka Connect REST API by enabling TLS encryption on the Kafka Connect REST listener.
```

Comment:
Does it really reduce access? Or just encrypt the connection?

Reply:
The "reduce access" phrase I reused from the previous proposal, but I agree changing to "The proposal adds the option to encrypt requests to the Kafka Connect REST API ..." would be clearer